packages/turf-ellipse/test/out/rotation-degrees.json

```diff
@@ -58,9 +54,7 @@
     {
       "type": "Feature",
       "bbox": [
-        -74.11421824800942,
-        40.64244465235168,
-        -73.8809366917385,
```

Comment:
we could maybe use printWidth of `90` for geojson and json files to avoid this awkward wrapping

Reply:
But maybe its just fine to leave it at 80, it usually wraps correctly and stuff like bigMatrix.json seems fine.

Reply:
Yeah this version with 80 looks better.  I'm open to it going higher for geojson and json as you said.

Reply:
I updated this with 90, if you're happy I'll plan on merging this one and closing #2670

Reply:
It looks fine to me.  Any further optimization can happen as follow-on